src/index.ts

```diff
@@ -15,6 +19,22 @@ router.get("/", async () => {
   });
 });
 
+router.put("/roles/create", createGuildRoleHandler);
+
+router.put("/roles/add", addGroupRoleHandler);
```

Comment:
will this be a put request?

Reply:
This is to keep it consistent with the HTTP method type of discord's endpoint.